submissions/Valerii49/side_menu/js/script.js

```diff
@@ -0,0 +1,151 @@
+"use strict";
+import sitesArr from '../data/attractions.js';
+
+document.addEventListener("DOMContentLoaded", InitPage(sitesArr));
+
+function InitPage(data) {
+	const menuList = document.querySelector('.menu__list');
+	const h1Link = document.querySelector('.h1__link');
+	//**********Init Sidebar**********
+	const menuLinks = data
+		.slice(1)
+		.map(({ id, sightName }) => `<li><a href="#" class="menu__link" data-id="${id}">${sightName}</a></li>`)
+		.join('');
+	menuList.innerHTML = menuLinks;
+	//**********for hover effect**********
+	const menuItems = document.querySelectorAll('.menu__link');
```

Comment:
Please use all document.querySelector methods in the top of your functions/conditions, 
it will be more structured and your code will be easier to understand